service.onidle/resources/settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Would be nice to migrate this to the new settings format at some point in the future